src/windeval/app.py

```diff
@@ -0,0 +1,72 @@
+import inspect
+
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import streamlit as st
+
+from windeval import diagnostics, load_product
+from windeval.processing import Diagnostics
+
+
+st.sidebar.markdown("# Load data")
+path1 = st.sidebar.text_input("Please enter path to the first data set.")
+path2 = st.sidebar.text_input("Please enter path to the second data set.")
+paths = [path1, path2]
+
+st.write("# Loaded data sets")
+fmap = {0: "First", 1: "Second"}
+for i in range(2):
+    if not Path(paths[i]).exists() or not paths[i]:
+        paths[i] = (
+            Path(__file__)
+            .resolve()
+            .parent.parent.parent.joinpath(
```

Comment:
Installing with `pip` in an otherwise empty conda env, this results in the app looking for data somewhere in the python standard lib.

Reply:
That was to be expected, unfortunate that the data is not copied there, I try to find an alternative, in the meantime, I'll allow the exists-check to fail.

Reply:
See the catalog repo for an example with included additional files: https://github.com/windeval/windeval-catalog/blob/master/pyproject.toml

Reply:
using the app in a "normal" environment should work now, it should just wait for you to input a valid data path before running the rest of the app.

We can adapt the behavior if desired.